dbtool.py

```diff
@@ -193,6 +193,48 @@ def dbshell(db: DbContext):
     sys.exit(retval)
 
 
+def exporttestdata(db: DbContext):
+    '''
+    This command must be run on a fully populated database, but the SQL
+    it generates can be useful for test suites and developers who want to
+    work on WoW without first processing lots of data.
+    '''
+
+    wow_table_name = 'wow_bldgs'
+    temp_table_name = 'temp_wow_bldgs_data_to_export'
+
+    # This is the business address for E&M Associates, reported in the
+    # New York Times as a prime example of a landlord who engages in
+    # aggresive eviction strategies to displace low-income tenants.
+    addr = '1465A FLATBUSH AVENUE 11210'
+
+    create_sql = (
+        f"create table {temp_table_name} as "
+        f"select * from {wow_table_name} where '{addr}' = Any(businessaddrs)"
+    )
+    drop_sql = f"drop table {temp_table_name}"
+
+    with db.connection() as conn:
+        cur = conn.cursor()
+        cur.execute(create_sql)
+
+    try:
+        env = os.environ.copy()
+        env['PGPASSWORD'] = db.password
+        output = subprocess.check_output([
+            'pg_dump',
+            '-h', db.host, '-p', str(db.port), '-U', db.user, '-d', db.database,
```

Comment:
This line and the environment are duplicated from the `dbshell` command so it'd be nice to keep things DRY here.

Reply:
Fixed this in b55e09b662113bb5857a14b053efa9ac496289b7!